salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -1,3 +1,5 @@
+interface: {{ salt_api_ip }}
```

Comment:
Whilst this is technically correct, maybe we shouldn't call this `salt_api_ip`, but `salt_ip` and rename the other occurrence as well?

Reply:
True